src/scaffoldmaker/meshtypes/meshtype_3d_nerve1.py

```diff
@@ -0,0 +1,1982 @@
+                if pole == 0:
+                    valueExpression.append((Node.VALUE_LABEL_D_DS2, [2]))
+                    d_ds1Expression.append((Node.VALUE_LABEL_D2_DS1DS2, [2]))
+                    d_ds2Expression.append((Node.VALUE_LABEL_D_DS3, [3]))
+                elif pole == 1:
+                    valueExpression.append((Node.VALUE_LABEL_D_DS3, [2]))
+                    d_ds1Expression.append((Node.VALUE_LABEL_D2_DS1DS3, [2]))
+                    d_ds2Expression.append((Node.VALUE_LABEL_D_DS2, [1, 3]))
+                elif pole == 2:
+                    valueExpression.append((Node.VALUE_LABEL_D_DS2, [1, 2]))
+                    d_ds1Expression.append((Node.VALUE_LABEL_D2_DS1DS2, [1, 2]))
+                    d_ds2Expression.append((Node.VALUE_LABEL_D_DS3, [1, 3]))
+                elif pole == 3:
+                    valueExpression.append((Node.VALUE_LABEL_D_DS3, [1, 2]))
+                    d_ds1Expression.append((Node.VALUE_LABEL_D2_DS1DS3, [1, 2]))
+                    d_ds2Expression.append((Node.VALUE_LABEL_D_DS2, [3]))
+
+                remapEftNodeValueLabel(eft2dNV, [ln], Node.VALUE_LABEL_VALUE, valueExpression)
+                remapEftNodeValueLabel(eft2dNV, [ln], Node.VALUE_LABEL_D_DS1, d_ds1Expression)
+                remapEftNodeValueLabel(eft2dNV, [ln], Node.VALUE_LABEL_D_DS2, d_ds2Expression)
+                ln += 2
+
+            remapEftLocalNodes(eft2dNV, 3, [1, 2, 3, 2])
+            facetemplate_branch_root = mesh2d.createElementtemplate()
+            facetemplate_branch_root.setElementShapeType(Element.SHAPE_TYPE_SQUARE)
+            facetemplate_branch_root.defineField(coordinates, -1, eft2dNV)
+            facetemplate_and_eft_list_branch_root[e] = (facetemplate_branch_root, eft2dNV)
+
+        # load data from file
+        print('Extracting data...')
+        vagus_data = load_vagus_data(region)
+        marker_data = vagus_data.get_level_markers()
+        trunk_data = vagus_data.get_trunk_coordinates()
+
+        invalid_data = False
+        if len(marker_data) < 2:
+            print("Missing or incomplete data. At least two landmarks are expected in the data.", file=sys.stderr)
+            invalid_data = True
+        if len(trunk_data) < 2:
+            print("Missing or incomplete data. At least two trunk points are required.", file=sys.stderr)
+            invalid_data = True
+        if vagus_data.get_side_label() not in ['left', 'right']:
+            print("Missing left or right marker side indication.", file=sys.stderr)
+            invalid_data = True
+        if invalid_data:
+            return [], None
+
+        if any(['level of esophageal hiatus' in marker_name or
+                'level of aortic hiatus' in marker_name for marker_name in marker_data.keys()]):
+            is_full_vagus = True
+            print('Estimate_trunk: Vagus top to bottom')
+        else:
+            is_full_vagus = False
+            print('Estimate_trunk: Vagus top to esophageal plexus')
```

Comment:
This would be better off being extrated to it's own method (minus the print statements).